### ISSUE TITLE
`triton create` should support assigning tags

### ISSUE THREAD

**Issue by bahamat** [joyent/node-triton#67]
Possibly also modifying tags on existing instances.

**Comment by trentm** [comment]
Showing it working (with coming commit):

```
$ cat bar.json
{
    "mynum": 42,
    "mybool": false
}

$ triton create -n foo0 minimal-64 t4-standard-256M -t foo=bar -t '{"fo":"baz"}' -t @bar.json -w
Creating instance foo0 (7be664cd-5e69-4259-9b67-1965b39ed697, minimal-64@15.3.0, t4-standard-256M)
Created instance foo0 (7be664cd-5e69-4259-9b67-1965b39ed697) in 32s

$ triton inst foo0 | json tags
{
  "foo": "bar",
  "fo": "baz",
  "mynum": 42,
  "mybool": false
}
```

**Comment by trentm** [comment]
published as triton@3.5.0

**Comment by trentm** [comment]
> Possibly also modifying tags on existing instances.

Ah. I haven't done that as `triton` doesn't have a command for updating an existing instance yet.
